src/java/edu/slu/tpen/transfer/JsonLDExporter.java

```diff
@@ -60,8 +61,76 @@ public class JsonLDExporter {
     * Populate a map which will contain all the relevant project information.
     *
     * @param proj the project to be exported.
+     * @param u  ? why'd it make me add these?
+     * @param profile  <--
     * @throws SQLException
+     * @throws IOException <--
     */
+   public void JsonLDExporter3(Project proj, User u, String profile) throws SQLException, IOException 
```

Comment:
Yes, right direction!  This can be `JsonLDExporter(Project proj, User u, String profile)` because of that extra parameter.  However, if you or the team specifically want a JsonLDExporter by a different name that's fine too.  I heard Mark say he would probably do content negotiation in ProjectServlet.  Here is an example usage that ProjectServlet might do (if it was named JsonLDExporter)...

```Java
.
.
.
String whichVersion = request.getVersionFromHTTPRequestHeaders()
if (whichVersion .equals("presi3")){
     resp.getWriter().write(new JsonLDExporter(proj, new User(uid), whichVersion).export());
     resp.setStatus(SC_OK);
}
else if (whichVersion.equals("presi2"){
     resp.getWriter().write(new JsonLDExporter(proj, new User(uid), whichVersion).export());
     resp.setStatus(SC_OK);
}
else{
    // The HTTP Request did not contain version information.  Do the default.
}
```